modules/local/checkm2/main.nf

```diff
@@ -1,13 +1,18 @@
 process CHECKM2 {
 
     label 'process_medium'
+    errorStrategy =
```

Comment:
Shouldn't this be in the modules.config file?

Reply:
I can add it there but I was not sure will nextflow take this errorStrategy or errorStrategy from process_medium from base.config...

Reply:
actually, those errorStrategies are the same as in base.config. I will remove them from modules at all :)